extension.js

```diff
@@ -139,6 +139,22 @@ function convertColor(color) {
     return undefined;
 }
 
+function syncStatusBarItemsWithActiveEditor() {
+    for (let statusBar of statusBarArray) {
+        if (!statusBar) {
+            continue;
+        }
+        statusBar.hide();
+        let currentFilePath = vscode.window.activeTextEditor && vscode.window.activeTextEditor.document.fileName;
+        if (!statusBar.filePattern || (currentFilePath && new RegExp(statusBar.filePattern).test(currentFilePath))) {
```

Comment:
RegExp may raise an exception and we should catch it.

Reply:
Thanks @actboy168, updated.